feat: mobile/desktop docs

SRWieZ opened the pull request:
<img width="1294" alt="Capture d’écran 2025-01-26 à 23 43 59" src="https://github.com/user-attachments/assets/8404f535-ddf4-4f56-b278-4df85ab06abd" />
<img width="1294" alt="Capture d’écran 2025-01-26 à 23 43 46" src="https://github.com/user-attachments/assets/49b4e5ea-3b88-4414-831f-bff44a81de38" />

simonhamp commented on routes/web.php:
```diff
@@ -23,7 +23,7 @@
 Route::get('/docs/{version}/{page?}', ShowDocumentationController::class)
     ->where('page', '(.*)')
-    ->where('version', '[0-9]+');
+    ->where('version', '[a-z0-9-]+');
```
I think we should have separate path parts:

/docs/desktop/1
/docs/mobile/1

And a redirect for the current URL pattern to /desktop/

SRWieZ replied:
Yep, didn't like it either. Thanks for your feedback. 

I'll make the adjustments this afternoon with redirections of the old desktop url.

simonhamp reviewed (changes requested):
I like the switcher. It may need to be simplified when viewing on mobile, but seems fine for now

My only challenge is with the URLs

SRWieZ commented:
Modified as per review.

Also:
-  Emptied the boilerplate of **mobile** documentation.
-  Created a new variable `$platform` everywhere `$version` was available.
-  Fixed a width issue when there was no text on a page.
-  Updated URLs from `docs/1/` to `docs/desktop/1/` in `*.md` files.
-  Put a **302** redirect on old URLs (you can change it to **301** when you want; it's just that 301 cache is so hard to clear, I preferred to wait for the full green light).
- Update SEO title to include platform and version

SRWieZ commented:
Reverted the following fix that made it worse in large screen:
- ~~Fixed a width issue when there was no text on a page (visible in screenshot above).~~

simonhamp commented:
This got merged manually
